qiskit/quantum_info/synthesis/two_qubit_decompose.py

```diff
@@ -161,7 +161,7 @@ def __init__(self, unitary_matrix):
         U *= la.det(U)**(-0.25)
 
         Up = _Bd.dot(U).dot(_B)
-        M2 = Up.T.dot(Up)
+        M2 = np.around(Up.T.dot(Up), 13)
```

Comment:
I see we use `np.around` a lot now in the code but it is a bit non-standard.  Would be nice to just do something like:

```python

if abs(M2) < 1e-15:
    M2 = 0
```
where `1e-15` can be replaced with any other `eps` value of ones choice.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/4835/commits/a6ab027af035b3cee40fedddda39745456e5cac8